classes/game.rb

```diff
@@ -0,0 +1,31 @@
+require_relative 'item'
+require 'date'
+
+##
+# Game class takes 2 arguments
+# @param multiplayer can two or more players play this game.
+# @param last_played_at define last time the game was played.
+# @param publish_date the date at which the game was published
+# @param archived (optional) to state if the game was already archived or not
+
+class Game < Item
+  attr_accessor :multiplayer, :last_played_at
+
+  def initialize(multiplayer, last_played_at, publish_date, archived: false)
+    super(publish_date, archived: archived)
+    @multiplayer = multiplayer
+    @last_played_at = last_played_at
+  end
+
+  private
+
+  def can_be_archived?
+    super && get_number_of_years(@last_played_at) > 2
+  end
```

Comment:
- [Optional] What do you think of doing it this way instead of having two methods?


```ruby
   def can_be_archived?
      current_year = Date.today.year
      two_years_ago = Date.new(current_year - 2)

      super && @last_played < two_years_ago
    end
```